### ISSUE TITLE
PPSSPP v1.8.0 can't read cheat Frontier Gate Boost+

### ISSUE THREAD

**Issue by endrare** [hrydgard/ppsspp#12057]

**Comment by endrare** [comment]
I tried the cheat game Frontier Gate Boost Plus iso v1.0.1 but it didn't work at ppsspp 1.8.0
screenshot: 
v1.8.0
https://ibb.co/Srkn5QB

Then I tried v1.7.5, v1.6.3, v1.5.3, v1.3.0 but it only works in v1.3.0
screenshot
v1.3.0
https://ibb.co/tBpP5zQ

**Comment by LunaMoo** [comment]
You're probably just placing the ini file into wrong folder, not sure about exact version, but in the distant past it was under PPSSPP/Cheats, it was moved into PPSSPP/memstick/PSP/Cheats or PSP/Cheats on the SD card. Also memstick folder does not have to be inside PPSSPP directory either, when running of a folder where it cannot write files, it will be automatically placed elsewhere depending on the operating system like in "my documents" on windows.

**Comment by endrare** [comment]
itu sudah benar tapi hanya bekerja di v1.3.0 untuk game lain tidak ada masalah
https://ibb.co/7zNNvbw

**Comment by endrare** [comment]
it is correct but only works in v1.3.0 for other games there is no problem
https://ibb.co/7zNNvbw

**Comment by LunaMoo** [comment]
Please upload that ini file.
 I see two possibilities, there's something within that ini formatting which doesn't work in Cheat menu anymore(possibly platform specific).
 Or something not related to cheats/cheat menu at all like if you store the game somewhere under PSP/Game folder and since it's a patched copy likely without original param.sfo the update param.sfo might not contain the info we're seeking which all commercial games have it might be detected as homebrew. If that fits your case, move the game backup to another location and it would work, because if it would be detected as homebrew, it would get a fake ID, through that would also be used by every other feature so for example save states would have different filenames as well, per-game config would also be affected etc.

**Comment by endrare** [comment]
.ini file
https://drive.google.com/file/d/1oIKv8aYB2dEopigjXzDrzWviXEEg1vLi/view?usp=drive_open

**Comment by LunaMoo** [comment]
Your google drive is private. Either make it public or just upload that file directly into your post here(might need to zip it first).

 Also where do you store that game, as I said if it's anywhere under "PSP/Game/" with non original param.sfo(patched game copy) this might cause issues and should be moved outside to a path which doesn't contain that string.

**Comment by endrare** [comment]
ah yeah, it turned out right after I moved it
outside the game folder it worked fine

**Comment by LunaMoo** [comment]
In that case I don't know if we can do anything about it outside of informing that Game folder should not be used to store commercial games.

 "PSP/Game" folder is special as on psp it was used only for dlc's, updates and homebrew. Some homebrew will only work from that folder and to enable ppsspp features like savestates we had to generate fakeID for them. We do it for all apps from that folder if they don't have specific entry in their param.sfo trying to avoid commercial games, unfortunately param.sfo from patched game like here might not have that and we generated fakeID for it anyway. It does't stop cheats from loading, it just uses different/fake ID for them, same with savestates, screenshots etc.. Not sure if we can find any better way to 100% detect homebrew/commercial games. In those old versions like 1.3, most features were broken for homebrew, so it was really bad.
